libs/docs/core/dialog/examples/dialog-mobile/dialog-mobile-example.component.ts

```diff
@@ -36,7 +37,8 @@ export class DialogMobileExampleComponent {
             mobile: true,
             responsivePadding: true,
             ariaLabelledBy: 'fd-dialog-header-5',
-            ariaDescribedBy: 'fd-dialog-body-5'
+            ariaDescribedBy: 'fd-dialog-body-5',
+            contentDensity: ContentDensityMode.COZY
```

Comment:
Doesn't appear to be overriding the app-level density:

![Screenshot 2024-09-11 at 9 23 00 AM](https://github.com/user-attachments/assets/aaeb165e-4420-446b-be68-9680baf0bb89)

Reply:
I reviewed the requirements and implementation again, and I think the expectations align with the requested requirements. Everything is still working as it was before. The only addition is the ability to configure content density, which was a requested requirement. Previously, users encountered issues setting fdCozy manually and had to use a workaround by setting mobile: true in the dialog configuration. This workaround transformed the header, footer, and other components to a cozy style but was not the desired approach, because fdcozy only was available on mobile. Now, users can configure different content densities directly within any dialog configuration.

It's important to note that when content density is set manually, it remains fixed and does not update with global content density changes. This behavior is consistent with how it worked previously in mobile mode as well.